src/scgpt/cross_check_genes/script.py

```diff
@@ -0,0 +1,70 @@
+import mudata as mu
+import numpy as np
+from scgpt.tokenizer.gene_tokenizer import GeneVocab
+
+## VIASH START
+par = {
+    "input": "resources_test/scgpt/test_resources/Kim2020_Lung_subset.h5mu",
+    "output": "output.h5mu",
+    "modality": "rna",
+    "input_var_gene_names": None,
+    "pad_token": "<pad>",
+    "vocab_file": "resources_test/scgpt/source/vocab.json"
+}
+## VIASH END
+
+# START TEMPORARY WORKAROUND setup_logger
+# reason: resources aren't available when using Nextflow fusion
+# from setup_logger import setup_logger
+def setup_logger():
+    import logging
+    from sys import stdout
+
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+    console_handler = logging.StreamHandler(stdout)
+    logFormatter = logging.Formatter("%(asctime)s %(levelname)-8s %(message)s")
+    console_handler.setFormatter(logFormatter)
+    logger.addHandler(console_handler)
+
+    return logger
+# END TEMPORARY WORKAROUND setup_logger
+logger = setup_logger()
+# Read in data
+logger.info(f"Reading {par['input']}")
+mudata = mu.read_h5mu(par["input"])
+adata = mudata.mod[par["modality"]].copy()
```

Comment:
Is this copy needed?

Reply:
isn't this what we do in all components?